src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewTextViewCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import UIKit
+
+class DynamicTableViewTextViewCell: UITableViewCell, DynamicTableViewTextCell {
+	private let textView = UITextView()
+
+	var dataDetectorTypes: UIDataDetectorTypes {
+		get { textView.dataDetectorTypes }
+		set { textView.dataDetectorTypes = newValue }
+	}
+
+	required init?(coder: NSCoder) {
+		super.init(coder: coder)
+		setup()
+	}
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setup()
+	}
+
+	private func setup() {
+		selectionStyle = .none
+		backgroundColor = .enaColor(for: .background)
+		textView.backgroundColor = .enaColor(for: .background)
+
+		textView.translatesAutoresizingMaskIntoConstraints = false
+		textView.isScrollEnabled = false
+		textView.isEditable = false
+		// The two below settings make the UITextView look more like a UILabel
+		textView.textContainerInset = .zero
+		textView.textContainer.lineFragmentPadding = .zero
+		textView.tintColor = .enaColor(for: .textTint)
```

Comment:
Why?

Reply:
I added a comment in the code.
Basically since these new `UITextView` backed cells may appear together with the normal `UILabel` type cells, we want to make sure that they look the same. `UITextView` has some additional insets that `UILabel` does not have. Not making these changes would cause some alignment issues, as can be seen below:

![IMG_D5D1DB98783F-1](https://user-images.githubusercontent.com/66173406/87190487-6a7b1680-c2a7-11ea-85f6-fe5d069d3cb4.jpeg)